frontend/src/components/VSvg/VSvg.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <SvgIcon :name="`images/${name}`" aria-hidden="true" focusable="false" />
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue"
+/**
+ * Displays the given SVG image.
+ */
+export default defineComponent({
+  name: "VIcon",
```

Comment:
Wait, is this `VIcon` or `VSvg`? 😮

Reply:
Great catch, left over from copy-pasting 😨